source/common/router/config_impl.cc

```diff
@@ -917,19 +922,20 @@ RouteEntryImplBase::getPathRewrite(const Http::RequestHeaderMap& headers,
   }
 
   // Return the regex rewrite substitution for redirects, if set.
-  if (regex_rewrite_redirect_ != nullptr) {
+  // redirect_config_ is known to not be nullptr here, because of the isRedirect check above.
```

Comment:
I think this can be added as an assert (so the invariant is not only stated in a comment).

Reply:
Done